CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ CHECK_FUNCTION_EXISTS(pthread_spin_init HAVE_PTHREAD_SPINLOCK)
 CHECK_FUNCTION_EXISTS(pthread_set_name_np HAVE_PTHREAD_SET_NAME_NP)
 CHECK_FUNCTION_EXISTS(pthread_setname_np HAVE_PTHREAD_SETNAME_NP)
 CHECK_FUNCTION_EXISTS(pthread_getname_np HAVE_PTHREAD_GETNAME_NP)
+CHECK_FUNCTION_EXISTS(pthread_rwlockattr_setkind_np HAVE_PTHREAD_RWLOCKATTR_SETKIND_NP)
```

Comment:
this sets a cmake variable `HAVE_PTHREAD_RWLOCKATTR_SETKIND_NP`

to make that visible to the compiler, we also need to add a `#cmakedefine` for it in `src/include/config-h.in.cmake`

Reply:
@cbodley ahh, i forgot that! fixed and repushed.